client/common/directives/DateTimePicker/DateTimePicker.js

```diff
@@ -32,7 +32,7 @@ angular.module('dashboard.directive.DateTimePicker', [
         elem.datetimepicker({
           format: scope.format,
           useCurrent: false,
-          defaultDate: scope.defaultDate ? moment(scope.defaultDate, scope.format) : undefined,
+          defaultDate: scope.defaultDate ? moment(scope.defaultDate, scope.format) : (scope.$parent.data.start ? scope.$parent.data.start : undefined),
```

Comment:
What is `$parent.data.start`? Is this custom for the project you are working on? If so, can you use the `defaultDate` property?